testing/test_motors.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import context
+import unittest
+import cv2
+import roboplot.core.stepper_motors as stepper_motors
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+import time
```

Comment:
Whilst it doesn't add much benefit for us since our project is small and we're familiar with the whole code base, I've been trying to put `import` statements in three groups:
  - The first for standard library imports
  - The second for third party imports
  - The third for `roboplot` imports

https://www.python.org/dev/peps/pep-0008/#imports